CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ option(ENABLE_PYTHON      "Enable python support?" ON)
 option(ENABLE_INTERNAL_FFMPEG "Enable internal ffmpeg?" OFF)
 if(UNIX)
   option(FFMPEG_PATH        "Path to external ffmpeg?" "")
+  option(ENABLE_INTERNAL_CDIO "Enable internal cdio?" ON)
```

Comment:
Why enable the internal version by default?

Reply:
because it's not optional

Reply:
That's not a reason for enabling the internal version by default.

Reply:
What's the point then?

Reply:
Disable it by default.